Fix the store prob with missing magic the most simple way

From: Thorsten Behrens <thb@openoffice.org>


---

 store/source/storbase.hxx |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)


diff --git store/source/storbase.hxx store/source/storbase.hxx
index ac75138..7d91545 100644
--- store/source/storbase.hxx
+++ store/source/storbase.hxx
@@ -701,7 +701,7 @@ class PageHolderObject
     template< class U >
     static bool isA (PageData const * p)
     {
-        return ((p != 0) && (p->type() == U::theTypeId));
+        return p != 0;
     }
 
     template< class U >
